src/libumf.map

```diff
@@ -2,7 +2,7 @@
 # Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-UMF_1.0 {
+UMF_0.10 {
```

Comment:
just want to confirm - didn't we say on some meeting, that it should be a bit diffrent name, so when we have the actual `1.0` version it won't overlap with this super old version used before...?